epaint/src/shape.rs

```diff
@@ -362,6 +378,57 @@ impl From<RectShape> for Shape {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+/// How rounded the corners of things should be
+pub struct Rounding {
+    pub nw: f32,
+    pub ne: f32,
+    pub sw: f32,
+    pub se: f32,
```

Comment:
```suggestion
    /// north-west = left-top
    pub nw: f32,
    /// north-east = right-top
    pub ne: f32,
    /// south-west = left-bottom
    pub sw: f32,
    /// south-east = right-bottom
    pub se: f32,
```

Reply:
Perhaps something like `/// Top left (north-west)` would be a bit better formatting wise for these?